toolchain/parse/handle_function.cpp

```diff
@@ -52,6 +52,30 @@ auto HandleFunctionSignatureFinish(Context& context) -> void {
       context.PushState(State::StatementScopeLoop);
       break;
     }
+    case Lex::TokenKind::Equal: {
```

Comment:
Is it fair to discuss whether assignment is the right syntax versus a modifier keyword due to the effects on parse complexity?

Reply:
I thought this approach would be simpler in the short term than adding support for modifiers with arguments, given where our current modifier support is. This approach also doesn't require adding any new token kinds. (From a design perspective rather than an implementation perspective, I also favor putting something after the function signature rather than before, because this is adding a definition for the function, and that's where we put definitions in general.)

But the goal here was just to add something that works for now, and explicitly not to block on working out what's the right long-term approach.